### DISCUSSION TITLE
How to set granular permissions without API

### DISCUSSION THREAD

**Discussion by azevedonik** [directus/directus#10672] [Archive]
Hi Guys, I was reading the section: https://docs.directus.io/reference/system/permissions/#permissions about creating permissions to allow only a certain field access by role and I'd like to know how can I create this inside the admin dashboard?

**Comment by rijkvanzanten** [comment]
Heya! You can navigate to [Settings > Roles & Permissions](https://docs.directus.io/configuration/users-roles-permissions/#configuring-permissions) and choose the role you want to configure.

From there, you can use the toggles to configure the permissions:

<img width="1345" alt="CleanShot 2021-12-24 at 20 44 03@2x" src="https://user-images.githubusercontent.com/9141017/147375607-6058b578-e8ea-4f1f-ab50-97a415e5bad7.png">

**Reply by azevedonik** [reply]
Thank you.
